src/Stratis.Bitcoin/Broadcasting/BroadcasterManagerBase.cs

```diff
@@ -11,25 +12,27 @@ public abstract class BroadcasterManagerBase : IBroadcasterManager
     {
         public event EventHandler<TransactionBroadcastEntry> TransactionStateChanged;
 
+        /// <summary>Connection manager for managing node connections.</summary>
+        protected readonly IConnectionManager connectionManager;
+
+        public BroadcasterManagerBase(IConnectionManager connectionManager)
+        {
+            this.connectionManager = connectionManager;
```

Comment:
You moved the `mempoolValidator` here but the check for null has disappeared.
Please reinstate this check by using `Guard.NotNull`.